core/src/main/scala/filodb.core/memstore/TimeSeriesMemStore.scala

```diff
@@ -145,12 +145,24 @@ extends MemStore with StrictLogging {
                     endOffset: Long,
                     checkpoints: Map[Int, Long],
                     reportingInterval: Long): Observable[Long] = {
+    var startOffsetValidated = false
```

Comment:
Just curious: Can this new var be declared within the first scope that needs it?

Reply:
done